common/protob/pb2py

```diff
@@ -65,18 +65,24 @@ INT_TYPES = (
 
 MESSAGE_TYPE_ENUM = "MessageType"
 
+LengthDelimited = c.Struct(
+    "len" / c.VarInt,
+    "bytes" / c.Bytes(c.this.len),
+)
+
 ListOfSimpleValues = c.GreedyRange(
     c.Struct(
         "key" / c.VarInt,
-        "value" / c.VarInt,
+        "value" / c.Switch(c.this.key & 0b111, {0: c.VarInt, 2: LengthDelimited}),
```

Comment:
Needed to not choke on the `java_package` and `java_outer_classname` file options.